institution/institution.go

```diff
@@ -0,0 +1,251 @@
+package institution
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+	"strconv"
+
+	"github.com/volatiletech/null/v8"
+	"github.com/volatiletech/sqlboiler/v4/boil"
+	"learningbay24.de/backend/models"
+)
+
+// GetAllUsers returns the number of registered users
+func GetUserCount(db *sql.DB) (int, error) {
+	// Check if there are more users in the course besides the creator
+	users, err := models.Users().Count(context.Background(), db)
+	if err != nil {
+		return -1, err
```

Comment:
Es ist egal, was man hier als den erten Rückgabewert zurückgibt, wenn ein error auch mitgegeben wird, aber die `-1` erinnert stark an C und wird in Go normalerweise einfach als der Default Wert von dem Datentyp zurückgegeben, bei int also `0`.
Musst du nicht ändern aber kannst du machen wenn du willst